dxr/build.py

```diff
@@ -447,7 +447,7 @@ def index_file(tree, tree_indexers, path, es, index):
     # will not accept the path, so just move on.
     rel_path = relpath(path, tree.source_folder)
     try:
-        rel_path = rel_path.decode('utf-8')
+        rel_path = unicode(rel_path, 'utf-8')
```

Comment:
By "config layer", I meant config.py. In general, it minimizes the opportunity for differing representations if we convert everything to unicode at the earliest possible point after it enters the system.